draft-ietf-mmusic-sctp-sdp.xml

```diff
@@ -122,14 +122,15 @@
       values with the SDP Offer/Answer mechanism <xref target="RFC3264"/> for negotiating
       SCTP-over-DTLS associations.
   	</t>
+    <t>
+      NOTE: Due to the characteristics of TCP, usage of 'TCP/DTLS/SCTP' will always
+      force ordered and reliable delivery of the SCTP packets, which limits the usage of
+      the SCTP options. Therefore, it is strongly RECOMMENDED that TCP is only used
+      in situations where UDP traffic is blocked.
+    </t>
 		<t>
-			NOTE: TLS is designed to run on top of a byte-stream oriented
-			transport protocol providing a reliable, in-sequence delivery like
-			TCP. <xref target="RFC6083"/> presents serious limitations with transporting TLS
-			on top of SCTP. Therefore, defining a mechanism to negotiate media streams
-			transported using TLS on top of SCTP, i.e., 'SCTP/TLS', is outside the scope
-			of this specification. In addition, defining a mechanism to negotiate non-protected
-      SCTP associations is also outside the scope of this specification.
+			NOTE: The definition of a mechanism to negotiate non-protected SCTP associations
```

Comment:
I then suggest to remove the note. Because, if we start listing other options, we'll probably have to add references etc for them too, and I see no advantage in doing that.